code/AssetLib/glTF2/glTF2Importer.cpp

```diff
@@ -1338,12 +1338,12 @@ std::unordered_map<unsigned int, AnimationSamplers> GatherSamplers(Animation &an
 
         auto& animsampler = anim.samplers[channel.sampler];
 
-        if (!animsampler.input) {
+        if (nullptr == animsampler.input) {
```

Comment:
This wasn't a nullptr check (as you can see from the build errors). The ref type has a bool() operator. You could do _if (false == animsampler.input)_ if you want it to be explicit.